script.js

```diff
@@ -0,0 +1,65 @@
+const roundingResult = 0.1 + 0.2
+console.log(roundingResult.toFixed(1));
+
+const userLogin = prompt('Enter your login');
+const userPassword = prompt('Enter your password');
+const login = 'admin';
+const password = '12pass33210';
+if (userLogin !== login && userPassword !== password) {
+    console.log('Wrong login and password!');
+} else if (userLogin !== login) {
+    console.log('Wrong login!');
+} else if (userPassword !== password) {
+    console.log('Wrong password!');
+} else {
+    console.log('Welcome!');
+}
+
+const amountOfMoney = +prompt('How much money do you have?');
+console.log('Price-list: watermalon: 21 uah; carrots: 8 uah; potatoes: 12 uah.');
+const watermalon = 21;
```

Comment:
```suggestion
const watermalonPrice = 21;
```
For better understating the value context it sounds more clear to me. The same for other prices
Plus, the values can be initialized as constants in uppercase.